WriterDesk1/src/pages/Progress.js

```diff
@@ -57,7 +57,7 @@ function Progress() {
     return (
         <>
             <div className='subTitle'>
-                <Typography variant='h5'>Average score per skill category</Typography>
+                <Typography variant='h5'>Average score of the last 5 documents</Typography>
```

Comment:
Ik denk dat het misschien beter is om dit een subtitel te maken, dus dat we als titel hebben: "Average score per skill category" en als subtitle: "This is using the feedback of the last 5 uploaded documents". Maar zo zou het ook kunnen I guess, mis je wel het per Skill category.